### ISSUE TITLE
php occ ldap:show-config json output

### ISSUE THREAD

**Issue by Yajo** [owncloud/user_ldap#592]
It would be great if one could do a `php occ ldap:show-config --output=json` command. Current outupt is quite hard to parse:

```
+-------------------------------+-------------+
| Configuration                 | s01         |
+-------------------------------+-------------+
| hasMemberOfFilterSupport      | 0           |
| homeFolderNamingRule          |             |
| lastJpegPhotoLookup           | 0           |
| ldapAgentName                 |             |
| ldapAgentPassword             | ***         |
| ldapAttributesForGroupSearch  |             |
| ldapAttributesForUserSearch   |             |
| ldapBackupHost                |             |
| ldapBackupPort                |             |
| ldapBase                      |             |
| ldapBaseGroups                |             |
| ldapBaseUsers                 |             |
| ldapCacheTTL                  | 600         |
| ldapConfigurationActive       | 0           |
| ldapDefaultPPolicyDN          |             |
| ldapDynamicGroupMemberURL     |             |
| ldapEmailAttribute            |             |
| ldapExperiencedAdmin          | 0           |
| ldapExpertUUIDGroupAttr       |             |
| ldapExpertUUIDUserAttr        |             |
| ldapExpertUsernameAttr        |             |
| ldapExtStorageHomeAttribute   |             |
| ldapGidNumber                 | gidNumber   |
| ldapGroupDisplayName          | cn          |
| ldapGroupFilter               |             |
| ldapGroupFilterGroups         |             |
| ldapGroupFilterMode           | 0           |
| ldapGroupFilterObjectclass    |             |
| ldapGroupMemberAssocAttr      |             |
| ldapHost                      |             |
| ldapIgnoreNamingRules         |             |
| ldapLoginFilter               |             |
| ldapLoginFilterAttributes     |             |
| ldapLoginFilterEmail          | 0           |
| ldapLoginFilterMode           | 0           |
| ldapLoginFilterUsername       | 1           |
| ldapNestedGroups              | 0           |
| ldapOverrideMainServer        |             |
| ldapPagingSize                | 500         |
| ldapPort                      |             |
| ldapQuotaAttribute            |             |
| ldapQuotaDefault              |             |
| ldapTLS                       | 0           |
| ldapUserAvatarRule            | default     |
| ldapUserDisplayName           | displayName |
| ldapUserDisplayName2          |             |
| ldapUserFilter                |             |
| ldapUserFilterGroups          |             |
| ldapUserFilterMode            | 0           |
| ldapUserFilterObjectclass     |             |
| ldapUuidGroupAttribute        | auto        |
| ldapUuidUserAttribute         | auto        |
| turnOffCertCheck              | 0           |
| turnOnPasswordChange          | 0           |
| useMemberOfToDetectMembership | 1           |
+-------------------------------+-------------+
```

**Comment by phil-davis** [comment]
It already works for me:
```
php occ ldap:create-empty-config
php occ ldap:show-config --output=json_pretty
{
    "hasMemberOfFilterSupport": "0",
    "hasPagedResultSupport": false,
    "homeFolderNamingRule": "",
    "lastJpegPhotoLookup": "0",
    "ldapAgentName": "",
    "ldapAgentPassword": "***",
    "ldapAttributesForGroupSearch": "",
    "ldapAttributesForUserSearch": "",
    "ldapBackupHost": "",
    "ldapBackupPort": "",
    "ldapBase": "",
    "ldapBaseGroups": "",
    "ldapBaseUsers": "",
    "ldapCacheTTL": "600",
    "ldapConfigurationActive": "0",
    "ldapDynamicGroupMemberURL": "",
    "ldapEmailAttribute": "",
    "ldapExperiencedAdmin": "0",
    "ldapExpertUUIDGroupAttr": "",
    "ldapExpertUUIDUserAttr": "",
    "ldapExpertUsernameAttr": "",
    "ldapGroupDisplayName": "cn",
    "ldapGroupFilter": "",
    "ldapGroupFilterGroups": "",
    "ldapGroupFilterMode": "0",
    "ldapGroupFilterObjectclass": "",
    "ldapGroupMemberAssocAttr": "uniqueMember",
    "ldapHost": "",
    "ldapIgnoreNamingRules": null,
    "ldapLoginFilter": "",
    "ldapLoginFilterAttributes": "",
    "ldapLoginFilterEmail": "0",
    "ldapLoginFilterMode": "0",
    "ldapLoginFilterUsername": "1",
    "ldapNestedGroups": "0",
    "ldapNetworkTimeout": "2",
    "ldapOverrideMainServer": "",
    "ldapPagingSize": "500",
    "ldapPort": "",
    "ldapQuotaAttribute": "",
    "ldapQuotaDefault": "",
    "ldapTLS": "0",
    "ldapUserDisplayName": "displayName",
    "ldapUserDisplayName2": "",
    "ldapUserFilter": "",
    "ldapUserFilterGroups": "",
    "ldapUserFilterMode": "0",
    "ldapUserFilterObjectclass": "",
    "ldapUserName": "samaccountname",
    "ldapUuidGroupAttribute": "auto",
    "ldapUuidUserAttribute": "auto",
    "turnOffCertCheck": "0",
    "useMemberOfToDetectMembership": "1"
}
```

**Comment by mmattel** [comment]
Closing, provided info from @phil-davis is correct
